tests/tests/eth_provider.rs

```diff
@@ -747,6 +755,35 @@ async fn test_send_raw_transaction(#[future] katana: Katana, _setup: ()) {
         .await
         .expect("failed to send transaction");
 
+    // Prepare the relayer
+    let relayer = katana.sequencer.account();
```

Comment:
should we maybe have the relayer be a field of the EOA? then you can get a relayer easily and just need to create the relayer at creation of the EOA instance?

Reply:
Yeah good idea just fixed